<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Personpage</class>
 <widget class="QWidget" name="Personpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="personalInfoTitle">
   <property name="geometry">
    <rect>
     <x>90</x>
<<<<<<< HEAD
     <y>40</y>
=======
     <y>100</y>
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>个人信息</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
<<<<<<< HEAD
     <y>70</y>
=======
     <y>130</y>
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="userNameTitle">
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="trackedCountTitle">
        <property name="text">
<<<<<<< HEAD
         <string>排名（被追踪次数）</string>
=======
         <string>被追踪次数</string>
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="successRateTitle">
        <property name="text">
<<<<<<< HEAD
         <string>资产（选股成功率）</string>
=======
         <string>选股成功率</string>
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
<<<<<<< HEAD
       <widget class="QLabel" name="userName">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="trackedCount">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="successRate">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
=======
       <widget class="QLineEdit" name="userNameEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="trackedCountEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="successRateEdit"/>
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
      </item>
     </layout>
    </item>
   </layout>
  </widget>
<<<<<<< HEAD
  <widget class="QLabel" name="introduceMessageTitle">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
=======
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>留言板</string>
   </property>
  </widget>
<<<<<<< HEAD
  <widget class="QLabel" name="holdingTitle">
=======
  <widget class="QLabel" name="label_2">
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>持仓情况</string>
   </property>
  </widget>
<<<<<<< HEAD
  <widget class="QLabel" name="tradeRecordTitle">
=======
  <widget class="QLabel" name="label_3">
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>270</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>交易记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
<<<<<<< HEAD
     <x>300</x>
     <y>410</y>
=======
     <x>310</x>
     <y>390</y>
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>点击查看持仓情况详情</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>540</x>
<<<<<<< HEAD
     <y>410</y>
=======
     <y>390</y>
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>点击查看交易记录详情</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="personalDescribeEdit">
   <property name="geometry">
    <rect>
<<<<<<< HEAD
     <x>20</x>
     <y>280</y>
     <width>191</width>
     <height>141</height>
=======
     <x>30</x>
     <y>320</y>
     <width>161</width>
     <height>111</height>
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="holdingsWidget" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>181</width>
<<<<<<< HEAD
     <height>111</height>
=======
     <height>91</height>
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QWidget" name="tradeWidget" native="true">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>290</y>
     <width>181</width>
<<<<<<< HEAD
     <height>111</height>
=======
     <height>91</height>
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
<<<<<<< HEAD
=======
  <widget class="QLabel" name="userAvatar">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>我是头像</string>
   </property>
  </widget>
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>451</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QLabel" name="balanceMonth">
      <property name="text">
       <string>本月资产</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
<<<<<<< HEAD
     <widget class="QLabel" name="phaseYearContent">
=======
     <widget class="QLabel" name="label_9">
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
<<<<<<< HEAD
     <widget class="QLabel" name="profitRateMonthContent">
=======
     <widget class="QLabel" name="label_4">
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
<<<<<<< HEAD
     <widget class="QLabel" name="operateCountYearContent">
=======
     <widget class="QLabel" name="label_11">
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
<<<<<<< HEAD
     <widget class="QLabel" name="profitRateYearContent">
=======
     <widget class="QLabel" name="label_8">
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="profitRateYear">
      <property name="text">
       <string>今年盈利率</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="profitRateTotal">
      <property name="text">
       <string>总盈利率</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="balanceYear">
      <property name="text">
       <string>今年资产</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
<<<<<<< HEAD
     <widget class="QLabel" name="balanceMonthContent">
=======
     <widget class="QLabel" name="label_6">
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="phaseYear">
      <property name="text">
       <string>今年盈亏</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="profitRateMonth">
      <property name="text">
       <string>本月盈利率</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="phaseMonth">
      <property name="text">
       <string>本月盈亏</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
<<<<<<< HEAD
     <widget class="QLabel" name="phaseMonthContent">
=======
     <widget class="QLabel" name="label_5">
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="operateCountTotal">
      <property name="text">
       <string>总操作数</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="balanceTotal">
      <property name="text">
       <string>总资产</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
<<<<<<< HEAD
     <widget class="QLabel" name="operateCountMonthContent">
=======
     <widget class="QLabel" name="label_7">
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="operateCountYear">
      <property name="text">
       <string>今年操作数</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="phaseTotal">
      <property name="text">
       <string>总盈亏</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="operateCountMonth">
      <property name="text">
       <string>本月操作次数</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
<<<<<<< HEAD
     <widget class="QLabel" name="balanceYearContent">
=======
     <widget class="QLabel" name="label_10">
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
<<<<<<< HEAD
     <widget class="QLabel" name="profitRateTotalContent">
=======
     <widget class="QLabel" name="label_12">
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
<<<<<<< HEAD
     <widget class="QLabel" name="phaseTotalContent">
=======
     <widget class="QLabel" name="label_13">
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
<<<<<<< HEAD
     <widget class="QLabel" name="balanceTotalContent">
=======
     <widget class="QLabel" name="label_14">
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
<<<<<<< HEAD
     <widget class="QLabel" name="operateCountTotalContent">
=======
     <widget class="QLabel" name="label_15">
>>>>>>> a9b89b36e241c5019cc123393d20395c9374dba1
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
